keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -312,16 +335,20 @@ you need any help or guidance.
 <!--
 This section must be completed when targeting alpha to a release.
 -->
+
 1.31:
+
 - enable by default
 - allow gate to disable the feature
 - release note
 
-1.32:
+1.34:
+
 - promote to GA
 - cannot be disabled
```

Comment:
This probably means `LockToDefault: true`. OK for me and remove in 1.35. The feature gate enables a flag disabled by default which is opt-in